src/Contract/Transaction.purs

```diff
@@ -25,17 +28,96 @@ module Contract.Transaction
 
 import Prelude
 
+import BalanceTx (BalanceTxError) as BalanceTxError
 import BalanceTx (UnattachedTransaction)
 import BalanceTx (balanceTx) as BalanceTx
-import BalanceTx (BalanceTxError) as BalanceTxError
+import Cardano.Types.Transaction
```

Comment:
Is there a reason to reexport so much from `Cardano.Types.Transaction`?

Reply:
I don't know, it comes from this commit:  e24723ff4971a43a650

However, somehow through merging the import ended up as a duplicate in my version, I will reduce it to one.